src/commands/friends/FriendCommand.ts

```diff
@@ -207,8 +235,8 @@ export default class FriendCommand extends Command {
   }
 }
 
-async function getApiUser (id: string): Promise<BotUser | undefined> {
-  const { data } = await axios.get(`${Plugin.config.api.address}/users/${id}?token=${Plugin.config.api.token}`)
+async function getApiUser (id: string): Promise<BotUser> {
```

Comment:
This shouldnt have been changed. It can return undefined.

What happens if you request `/users/99999999999`? It would 404.

Reply:
oops